instrumentation/spring/spring-ws-2.0/javaagent/src/test/java/test/boot/SpringWsTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.boot;
```

Comment:
for java tests our convention is to place them in the same package as the instrumentation so here the test code should be in `io.opentelemetry.javaagent.instrumentation.spring.ws.v2_0`